azure-pipelines.yml

```diff
@@ -142,29 +141,32 @@ jobs:
         python.version: '3.9'
   steps:
   - checkout: self
-    submodules: true
-    condition: and(succeeded(), or(eq(variables.buildWheel, true), eq(variables['python.version'], '3.9')))
+    submodules: True
+    condition: and(succeeded(), or(eq(variables.buildWheel, True), eq(variables['python.version'], '3.9')))
   - script: |
-      if [ "$(buildWheel)" == "true" ]; then
+      if [ "$(buildWheel)" == "True" ]; then
         export TAG="-nightly"
       else
         export TAG=""
       fi
-      echo "TAG=$TAG"
       docker run --rm \
         -w /root/nmodl \
         -v $PWD:/root/nmodl \
         -e NMODL_NIGHTLY_TAG=$TAG \
         'bluebrain/nmodl:wheel' \
         packaging/build_wheels.bash linux $(python.version)
-    condition: and(succeeded(), or(eq(variables.buildWheel, true), eq(variables['python.version'], '3.9')))
+    condition: and(succeeded(), or(eq(variables.buildWheel, True), eq(variables['python.version'], '3.9')))
     displayName: 'Building ManyLinux Wheel'
+  #- task: PublishBuildArtifacts@1
+  #  inputs:
+  #    pathToPublish: '$(Build.SourcesDirectory)/wheelhouse'
+  #  displayName: 'Publish wheel as build artifact'
```

Comment:
I would uncomment this

Reply:
You want to always produce artefacts?

Reply:
yes, that's we typically do. Artefacts get deleted automatically after certain time by Azure automatically .